pyproject.toml

```diff
@@ -34,7 +34,7 @@ readme = "README.md"
 # Investigate if it is better to fix specific versions or use lower and upper bounds  
 # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/2665
 python = ">=3.8.1,<3.11"
-concrete-python = "2.5.0-rc1 "
+#concrete-python = "2023.11.5"
```

Comment:
is it right to comment this ? why not keeping the`2.5.0-rc1 ` version to make poetry consider more realistic constraint in the dependencies to install ? this is an open question, not sure of this answer

Reply:
Not sure how well pypi handles nightly but maybe we could publish the nightlies that we use to pypi if there is no IP blocker.

Reply:
I think that's the main issue with nightly, I believe we don't want them to be public, only rc should

Reply:
Maybe we should them for a rc once we see that the nightly is working for us?